### ISSUE TITLE
Modifie CSV output format

### ISSUE THREAD

**Issue by swilde** [Intevation/intelmq-mailgen#33]
As per cutomer request:

CSV output should: 
- be seperated by commas
- quote every field using quotation marks

I'll implement the change.

**Comment by bernhardreiter** [comment]
Personally I guess that unnecessary quotes will make a file less readable.
What is the rationale for quoting "all fields"?

**Comment by swilde** [comment]
I agree that this reduces human readability.  It was suggested by the
customer to simplify the format (by making it more consistent) in
order to make writing parser as easy as possible.

Regarding the readability the focus has changed over the course:

- Originally the format as used in the customers legacy system was
  some pseudo csv, which was indeed optimized for reading by humans:

  It used pipe symbols "|" as field separator and aligned fields by
  padding them with spaces to get a nice tabular display:

    1234 | 10.0.1.2  | Example | 80 | tcp | foobar
    1235 | 10.0.8.15 | More    | 80 | tcp | buzz

  The Details of quoting and escaping (if any) are unknown to us for
  this format.

- As the specification said that the format should be csv (which the
  padded version really wasn't) and emulating the old behavior would
  have been expensive (compared to using a standard python csv
  encoding library) we suggested to use a normalized csv format, still
  with the "|" separator, and with " for quoting:

    1234|10.0.1.2|Example|80|tcp|foobar
    1235|10.0.8.15|More|80|tcp|buzz
    1236|10.1.2.3|"Quotes ("") example, with pipe|"|80|tcp|buzz

- As this already lost much of the original readability the focus was
  switched to easy automatic parsing.  The format should be as uniform
  and easy to parse as possible and match expectations.  That's why
  the delimiter was changed to comma (matching the "c" in "csv"), and
  everything is quoted:

    "1234","10.0.1.2","Example","80","tcp","foobar"
    "1235","10.0.8.15","More","80","tcp","buzz"
    "1236","10.1.2.3","Quotes ("") example, with pipe|","80","tcp","buzz"

I'm not sure whether the "quote everything" really is an improvement,
but I'm sure it really doesn't hurt either.
